### PR TITLE
Navbar/shannon

### PR DESCRIPTION
**Pull request by shannon97**
## Describe your changes
SO - started setup of navbar foundation with header as global element

## Type of change
- [ ] Bug fix (non-breaking change which fixes an issue)
- [X] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## What kind of feedback do you need?
- [X] Make sure code works and advise if I have missed anything
- [ ] No other major issues to review

## Checklist:
- [x]  I have performed a self-review of my code
- [X]  I have recommended where to start and how to proceed with the review

### CODE REVIEW

File: src/components/GlobalElements/NavBar.jsx
Comment (caoimhejyoti) [inline src/components/GlobalElements/NavBar.jsx:1]: OBSESSED!

**Review by caoimhejyoti — APPROVED** [review]
Smashing GOALS!

File: src/components/GlobalElements/NavBar.jsx
Comment (BBLogan) [reply src/components/GlobalElements/NavBar.jsx:1]: <3 :D

**Review by BBLogan — APPROVED** [review]
Looks good to me team - great work so far - keep it up! Without burning out pleasssseeeee 🥇 👍 :)

**Pull request merged**